pybikes/data/gbfs.json

```diff
@@ -907,6 +907,20 @@
             },
             "feed_url":"https://saint-etienne-gbfs.klervi.net/gbfs/en/gbfs.json"
         },
+        {
+            "tag":"velopop",
+            "meta":{
+                "city":"Avignon",
+                "name":"V\u00e9lopop",
+                "country":"FR",
+                "company": [
+                    "Smoove"
+                ],
+                "latitude":43.943689,
+                "longitude":4.805833
+            },
+            "feed_url":"https://avignon-gbfs.klervi.net/gbfs/en/station_information.json"
```

Comment:
the `feed_url` should be `https://avignon-gbfs.klervi.net/gbfs/en/gbfs.json` to match the criteria of the [gbfs.py](https://github.com/eskerda/pybikes/blob/97414d1c9f3ebef6cf96e8637b15a77820d32612/pybikes/gbfs.py) parser.